client/components/download-link/renderers/docx-renderer.js

```diff
@@ -627,7 +644,7 @@ export default (application, sections, values, updateImageDimensions) => {
           if (step.singular) {
             doc.createParagraph(`${step.singular} ${index + 1}`).heading2();
           }
-          (step.fields || []).filter(f => f.repeats).forEach(field => renderField(doc, field, v, project));
+          (step.fields || []).forEach(field => renderField(doc, field, v, project));
```

Comment:
Do we know why has this filter been removed?

Reply:
Nope.

Reply:
Paging @joefitter

Reply:
:wave

Reply:
As per the schema, no fields have repeats and we ended up with the fields themselves not being rendered.